website/source/docs/builders/hyperv-iso.html.markdown

```diff
@@ -0,0 +1,982 @@
+---
+description: |-
+  The HyperV Packer builder is able to create HyperV virtual machines and export them.
+layout: "docs"
+page_title: "HyperV Builder (from an ISO)"
+---
+
+# HyperV Builder (from an ISO)
+
+Type: `hyperv-iso`
+
+The HyperV Packer builder is able to create [HyperV](https://www.microsoft.com/en-us/server-cloud/solutions/virtualization.aspx)
+virtual machines and export them, starting from an ISO image.
+
+The builder builds a virtual machine by creating a new virtual machine
+from scratch, booting it, installing an OS, provisioning software within
+the OS, then shutting it down. The result of the HyperV builder is a directory
+containing all the files necessary to run the virtual machine portably.
+
+## Basic Example
+
+Here is a basic example. This example is not functional. It will start the
+OS installer but then fail because we don't provide the preseed file for
+Ubuntu to self-install. Still, the example serves to show the basic configuration:
+
+```javascript
+{
+  "type": "hyperv-iso",
+  "guest_os_type": "Ubuntu_64",
+  "iso_url": "http://releases.ubuntu.com/12.04/ubuntu-12.04.5-server-amd64.iso",
+  "iso_checksum": "769474248a3897f4865817446f9a4a53",
+  "iso_checksum_type": "md5",
+  "ssh_username": "packer",
+  "ssh_password": "packer",
+  "shutdown_command": "echo 'packer' | sudo -S shutdown -P now"
+}
+```
+
+It is important to add a `shutdown_command`. By default Packer halts the
+virtual machine and the file system may not be sync'd. Thus, changes made in a
+provisioner might not be saved.
+
+## Configuration Reference
+
+There are many configuration options available for the HyperV builder.
+They are organized below into two categories: required and optional. Within
+each category, the available options are alphabetized and described.
+
+In addition to the options listed here, a
+[communicator](/docs/templates/communicator.html)
+can be configured for this builder.
+
+### Required:
+
+-   `iso_checksum` (string) - The checksum for the OS ISO file. Because ISO
+    files are so large, this is required and Packer will verify it prior
+    to booting a virtual machine with the ISO attached. The type of the
+    checksum is specified with `iso_checksum_type`, documented below.
+
+-   `iso_checksum_type` (string) - The type of the checksum specified in
+    `iso_checksum`. Valid values are "none", "md5", "sha1", "sha256", or
+    "sha512" currently. While "none" will skip checksumming, this is not
+    recommended since ISO files are generally large and corruption does happen
+    from time to time.
+
+-   `iso_url` (string) - A URL to the ISO containing the installation image.
+    This URL can be either an HTTP URL or a file URL (or path to a file).
+    If this is an HTTP URL, Packer will download it and cache it between
+    runs.
+
+### Optional:
```

Comment:
This section seems to be missing a lot of configuration _options_, `ram_size_mb`, `guest_addition*` etc.